src/diagonal.works/b6/api/histogram.go

```diff
@@ -48,36 +56,123 @@ func NewHistogramFromCollection(c b6.UntypedCollection, id b6.CollectionID) (*in
 	return &histogram, nil
 }
 
-func HistogramBucketLabels(c b6.CollectionFeature) []string {
-	i := 0
-	labels := make([]string, 0)
+func newOriginDestinationHistogram(c b6.UntypedCollection) (*ingest.CollectionFeature, error) {
+	histogram := ingest.CollectionFeature{
+		Tags: []b6.Tag{{Key: "b6:od", Value: b6.String("yes")}},
```

Comment:
i mean please do better than b6:od / unless that makes ur life hell rn somehow

Reply:
also const it i ve made my life miserable w tag key typos a few times

Reply:
Done.